roles/kube_nginx_ingress_monitoring/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+- name: 'create addons dir if needed'
```

Comment:
I'm confused now. How does this relate to `roles/kube_nginx_ingress/tasks/prometheus_nginx.yml`?

Reply:
It's a move but it did `mv` instead of `git mv`